python/cugraph-pyg/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+[build-system]
+
+requires = [
+    "wheel",
+    "setuptools",
+    "cython>=0.29,<0.30",
+    "scikit-build>=0.13.1",
+    "cmake>=3.23.1",
+    "ninja",
```

Comment:
I dont think the build requires all of these ?

Reply:
dropped cython and cmake